web/admin/process.ex

```diff
@@ -65,6 +67,27 @@ defmodule Vae.ExAdmin.Process do
       end
     end
 
+    def duplicate_action(conn, params) do
+      Process
+      |> Repo.get(params[:id])
+      |> Repo.preload(:delegates)
+      |> Repo.preload(:processes_steps)
```

Comment:
use a list of atoms to preload multiple associations: `Repo.preload([:delegates, :processes_steps])`